src/main/java/Duke.java

```diff
@@ -6,5 +11,87 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greetLine();
+        addList();
+        exitLine();
+    }
+    public static void greetLine(){
+        System.out.println("How may I be of service?");
+    }
+    public static void echo()
+    {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")) {
+            System.out.println(line);
+            line = in.nextLine();
+        }
+    }
```

Comment:
You'd want to indent the { } properly, like:
public static void echo(){

// your code
 }